cns/restserver/restserver.go

```diff
@@ -1174,6 +1179,9 @@ func (service *HTTPRestService) createOrUpdateNetworkContainer(w http.ResponseWr
 
 	reserveResp := &cns.CreateNetworkContainerResponse{Response: resp}
 	err = service.Listener.Encode(w, &reserveResp)
+
+	logNCSnapshot(req)
```

Comment:
We should only send the snapshot if this NC request succeeds. In this case, we are sending the snapshot regardless.

Reply:
Good point. Changed to check for the return code and log snapshot only when successful.